src/examples/demo_face_detector.cpp

```diff
@@ -25,7 +25,7 @@
 using namespace cv;
 using namespace std;
 using json = nlohmann::json;
-using namespace daisykit::common;
+using namespace daisykit::types;
 using namespace daisykit::flows;
```

Comment:
```suggestion
namespace daisykit {
```

Reply:
We should use namespace daisykit and add names of submodule before their corresponding declarations.

Reply:
Apply for all